packages/react-components/src/types/ParticipantListParticipant.ts

```diff
@@ -32,6 +35,23 @@ export type RaisedHand = {
   raisedHandOrderPosition: number;
 };
 
+/* @conditional-compile-remove(reaction) */
+/**
+ * Reaction state with reaction type to render
+ *
+ * @public
+ */
+export type Reaction = {
+  /**
+   * Specifies the type of reaction videoTile should render i.e. like, heart etc.
+   */
+  reactionType: string;
+  /**
+   * Received timestamp relative to 01-01-2020 0:0:0 hours

```

Comment:
Why is the timestamp relative to `01-01-2020 0:0:0 hours`? Can't it just be any `Date`?